### PR TITLE
v0.1.13 release

### PR DESCRIPTION
**Pull request by gregtatcam**
added domain configuration

### CODE REVIEW

File: packages/imaplet-lwt/imaplet-lwt.0.1.13/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+authors: "Gregory Tsipenyuk <gregtsip@gmail.com>"
+license: "MIT"
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+remove: [
+  ["ocamlfind" "remove" "imaplet"]
+  ["rm" "-f" "%{bin}%/imaplet*"]
```
Comment (avsm) [inline packages/imaplet-lwt/imaplet-lwt.0.1.13/opam]: would it be possible to change this wildcard into an explicit set of rm commands?  It's a bit unsafe to just remove anything called imaplet

Comment (gregtatcam) [inline packages/imaplet-lwt/imaplet-lwt.0.1.13/opam]: Fixed in the next release

Comment (Chris00) [inline packages/imaplet-lwt/imaplet-lwt.0.1.13/opam]: Alternatively, you can add an `install` file — [oasis2opam](https://github.com/ocaml/oasis2opam) will generate one for you.  If you use [oasis2opam](https://github.com/ocaml/oasis2opam), consider adding the version constraints to `_oasis`.

Comment (gregtatcam) [inline packages/imaplet-lwt/imaplet-lwt.0.1.13/opam]: Cook, thanks. I'll take a look.

**Pull request merged**